plugins/Editing.jsx

```diff
@@ -233,10 +264,30 @@ class Editing extends React.Component {
             </SideBar>
         );
     }
+
+    checkVisibility = (selectedLayer, visibility) => {
+        if (selectedLayer != null){
+            let layer = this.getLayer(this.props.layers)
```

Comment:
This won't work if the theme layer is split (i.e. when reordering is active an an external layer is placed in the middle).

You'll want to do something like

    let path = null;
    this.props.layers.filter(layer => (layer.role === LayerRole.THEME && LayerUtils.searchSubLayer(layer, 'name', selectedLayer, path));

and you'll also get the path for free.